app/mm-redux/utils/channel_utils.ts

```diff
@@ -629,6 +630,17 @@ export function filterChannelsMatchingTerm(channels: Array<Channel>, term: strin
     });
 }
 
+export function removeItem<T>(array: T[], item: T) {
```

Comment:
Now that it is declared in array utils, can we get rid of this one? Or is this still needed?